docs/repo/property-pages/how-to-add-a-new-launch-profile-kind.md

```diff
@@ -103,6 +103,125 @@ And you're done. Projects that import the .targets file will now show this UI wh
 
 ## Option 2: Embedded XAML file
 
-In this option the XAML file is embedded in an assembly as a resource and discovered by means of a MEF export. Compared to Option 1 this requires more initial setup but does not require you to distribute an additional file. This may be an attractive option if you are already exporting MEF components for use in Visual Studio.
+In this option the XAML file is embedded in an assembly as a resource and discovered by means of a MEF export. Compared to Option 1, this requires more initial setup but does not require you to distribute an additional file. This may be an attractive option if you are already exporting MEF components for use in Visual Studio.
 
-_Steps to be determined._
+### Step 1: Define the `XamlPropertyRule` item
+
+Add the following item to your .csproj file:
+
+```xml
+  <ItemGroup>
+    <XamlPropertyRule Include="path\to\MyPropertyPage.xaml">
+      <Namespace>MyNamespace</Namespace>
+      <DataAccess>IRule</DataAccess>
+      <RuleInjectionClassName>MyPropertyPageProperties</RuleInjectionClassName>
+    </XamlPropertyRule>
+  </ItemGroup>
+```
+
+### Step 2: Register the property page and properties
+
+Add a partial class that corresponds to the properties defined within the rule and generated during build:
+
+```csharp
+using Microsoft.VisualStudio.ProjectSystem;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+using System;
+using System.ComponentModel.Composition;
+
+namespace MyNamespace
+{
+    [Export]
+    [AppliesTo(MyProject.UniqueCapability)]
+    internal partial class MyPropertyPageProperties : StronglyTypedPropertyAccess
+    {
+        [ExportPropertyXamlRuleDefinition("<Assembly Name>, Version=<Assembly Version>, Culture=<Assembly Culture>, PublicKeyToken=<Assembly Public Key Token>", "XamlRuleToCode:MyPropertyPage.xaml", "Project")]
```

Comment:
Maybe call out that this can be simplified using an approach similar to that used in this repo's `ExportRuleAttribute`.

Reply:
This is still I good idea, but I ended up leaving it off of this how-to for the sake of brevity. We should find a different place for this, though.